src/module/data/actor/character.mjs

```diff
@@ -23,13 +23,13 @@ export default class CharacterModel extends BaseActorModel {
     const schema = super.defineSchema();
 
     schema.hero = new fields.SchemaField({
-      // Some classes have a second resource
       primary: new fields.SchemaField({
         value: new fields.NumberField({initial: 0, min: 0, integer: true, nullable: false})
       }),
-      secondary: new fields.SchemaField({
-        value: new fields.NumberField({initial: null, min: 0, integer: true})
-      }),
+      // Possible future expansions will have classes with multiple resources
```

Comment:
In v13, maybe this can make use of the TypedSchemaField or whatever name they decide on for the MappingField.

Reply:
It does look like for level 10 characters (in the Patreon packet) they're experimenting with "epic" resources, but we'll see if those survive this round of playtesting.